dataactvalidator/validation_handlers/validator.py

```diff
@@ -375,12 +374,12 @@ def rule_greater(cls, data, value, rule, datatype, interfaces, record):
     @classmethod
     def rule_equal(cls, data, value, rule, datatype, interfaces, record):
         """Checks that data is equal to specified value"""
-        return Validator.getType(data,datatype) == Validator.getType(value,datatype)
+        return Validator.getType(data.lower(),datatype) == Validator.getType(value.lower(),datatype)
```

Comment:
No, not always. But `data` is a string until it's converted using getType so calling `lower()` on that should be fine.